src/main/scala/com/lightbend/lagom/docs/DocumentationGenerator.scala

```diff
@@ -33,7 +33,7 @@ object DocumentationGenerator extends App {
   )
 
   val previewVersions = Seq(
-    VersionSummary("1.5.x", s"Lagom 1.5.0-M3 (preview)")
+    VersionSummary("master", s"Lagom 1.5.0-M3 (preview)")
```

Comment:
Before the change that introduced `1.5.x` as a preview version, this list was empty.

`master` wasn't considered a preview

Reply:
I think these `name`s are branch names (not tags). I don't think we0're sticking to a tag in any of the branches.

Reply:
We're using `master` because we still haven't cut `1.5.x`. As soon as we cut it we will change this. In general `master` is _not_ preview, I agree.

Reply:
Sure, and we don't generate the docs for a tag, only branch. 

So either we link to `master` and we have the risk to have a documentation of something that can't yet be used (eg: LagomClientFactory new API) or we change the site generation to also support tags. 

I stick with option 1, we don't have the bandwidth for option 2